src/rgen/Microsoft.Macios.Transformer/TransformCommand.cs

```diff
@@ -0,0 +1,38 @@
+using System.CommandLine;
+
+namespace Microsoft.Macios.Transformer;
+
+public class TransformCommand : Command {
+
+	public TransformCommand () : base ("rgen-transform", "command to convert outdated bindings to be rgen compatible")
+	{
+		var input = new Option<string> (["--input", "-i"], "input directory to search for bgen bindings") {
```

Comment:
```suggestion
		var input = new Option<string> (["--input", "-i"], "input directory to search for rgen bindings") {
```

typo?

Reply:
no, this was to reference the previosuly generated bindings